src/core/scatter_path.gd

```diff
@@ -57,15 +57,17 @@ func get_pos_and_normal(offset) -> Array:
 	var normal := Vector3.ZERO
 	
 	var pos1
-	if offset + 0.1 < curve.get_baked_length():
-		pos1 = curve.interpolate_baked(offset + 0.1)
+	if offset + curve.get_bake_interval() < curve.get_baked_length():
+		pos1 = curve.interpolate_baked(offset + curve.get_bake_interval())
 		normal = (pos1 - pos)
 	else:
-		pos1 = curve.interpolate_baked(offset - 0.1)
+		pos1 = curve.interpolate_baked(offset - curve.get_bake_interval())
 		normal = (pos - pos1)
-
-	normal.y = 0.0
-	normal = normal.normalized().rotated(Vector3.UP, PI / 2.0)
+	
+	#not sure what these lines are intended to do, but they break utility
+	#ommitting the normals y axis is silly, probably the root cause of why alignment wasn't working
+#	normal.y = 0.0
+#	normal = normal.normalized().rotated(Vector3.UP, PI / 2.0)
```

Comment:
Yes you can remove all that, it was from the previous version where the curve was always constrained to a plane so the Y axis was always 0 anyway.